src/utils/typed-array-utils.js

```diff
@@ -35,7 +35,7 @@ export function getGLTypeFromTypedArray(arrayOrType) {
 // Used to auto deduce gl parameter types
 
 /* eslint-disable complexity */
-export function getTypedArrayFromGLType(glType, {clamped = true} = {}) {
+export function getTypedArrayFromGLType(glType, {clamped = false} = {}) {
```

Comment:
Why set this to false by default? Can't we specify the param in readPixels instead? A little worried that we break something else...